### PR TITLE
Real time

### PR DESCRIPTION
**Pull request by thaonp104**
## Related Tickets:
- [29105](https://edu-redmine.sun-asterisk.vn/issues/29105)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.
ezgif-4-bb709552ca8c

### CODE REVIEW

**Comment by thaonp104** [comment]
ready

File: app/Http/Controllers/NotificationController.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            ]);
+        }
+
+        return redirect()->route($routeName, $idItem);
+    }
+
+    public function getNotifications()
+    {
+        $notifications = Auth::user()->notifications;
+        return $notifications;
```
Comment (huyenttm-0901) [inline app/Http/Controllers/NotificationController.php]: convention

File: app/Notifications/TraineeNotification.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        return ['database'];
+    }
+
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->line('The introduction to the notification.')
+            ->action('Notification Action', url('/'))
+            ->line('Thank you for using our application!');
```
Comment (huyenttm-0901) [inline app/Notifications/TraineeNotification.php]: i18n, url

**Comment by thaonp104** [comment]
ready

**Pull request merged**